parm/archive/master_gefs.yaml.j2

```diff
@@ -0,0 +1,12 @@
+# Set variables/lists needed to parse the gefs templates
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+
+datasets:
+{% filter indent(width=4) %}
+    {% include "gefs_extracted_atmos.yaml.j2" %}
+    {% include "gefs_extracted_ocean.yaml.j2" %}
+    {% include "gefs_extracted_ice.yaml.j2" %}
+    {% include "gefs_extracted_wave.yaml.j2" %}
```

Comment:
> `workflow/applications/gefs.py` should be updated to only run archive when `do.extractvars` is `true`. Otherwise I think this is ready.

Given the current yaml files, the archive job should succeed without a `DO_EXTRACTVARS` condition for base GEFS (based on my tests and Anton's test according to our discussions). However, since the GEFS reforecast archived products will likely be structured differently than the GEFS retrospective products, it may be best to use a flag such as `DO_EXTRACTVARS` for the reforecast products. 

If we use `DO_EXTRACTVARS`, we may need yaml files for base GEFS (see suggestion below) so that the CI test doesn't fail. In the CI test, `DO_EXTRACTVARS=NO` and `HPSSARCH=YES`, which I think will cause the archive task to fail if there are no data files to stage. 

```suggestion
{% if DO_EXTRACTVARS %}
    {% filter indent(width=4) %}
    {% include "gefs_extracted_atmos.yaml.j2" %}
    {% include "gefs_extracted_ocean.yaml.j2" %}
    {% include "gefs_extracted_ice.yaml.j2" %}
    {% include "gefs_extracted_wave.yaml.j2" %}
    {% endfilter %}
{% else %}
    {% filter indent(width=4) %}
    {% include "gefs_atmos.yaml.j2" %}
    {% include "gefs_ocean.yaml.j2" %}
    {% include "gefs_ice.yaml.j2" %}
    {% include "gefs_wave.yaml.j2" %}
    {% endfilter %}
{% endif %}
```